packages/firebase_database/android/src/main/java/io/flutter/plugins/firebase/database/FirebaseDatabasePlugin.java

```diff
@@ -246,6 +246,14 @@ public void onMethodCall(MethodCall call, final Result result) {
           break;
         }
 
+      case "DatabaseReference#update":
+      {
```

Comment:
I think you need to add `Map<String, Object> arguments = call.arguments();` here